.gitignore

```diff
@@ -12,3 +12,6 @@ npm-debug.log
 
 # Netlify
 .netlify
+
+#VSCode settings; makes life easier for some, tougher for most
+.vscode/
```

Comment:
If you run VSCode, you should probably put this in a .gitignore file in your home directory. I'd prefer we don't try to accommodate every IDE separately in our repo .gitignore.

Reply:
Removed in 956b1ac